src/resourceGenUtils.js

```diff
@@ -185,6 +167,18 @@ module.exports = {
   getPackageInfo: (dir = cwd) => fs.readJson(
     path.resolve(dir, 'package.json')),
 
+  /**
+   * Copies devDependencies from this project to the new project
+   */
+  copyPackages: () => {
+    console.log('copying packages...');
+    Object.keys(devDependencies).forEach((m) => {
+      const fromDir = path.resolve(__dirname, '..', 'node_modules', m);
+      const toDir = path.resolve(cwd, 'node_modules', m);
+      fs.copySync(fromDir, toDir);
```

Comment:
Interesting concept. I can see how this could be faster than letting "npm install" do the download, but does it run the risk of modules being out of date in the new project?

Reply:
It does, technically. I was originally also doing an npm update before the copy to prevent that, I could add that back in if you think its important. I took it out because it was still taking longer than I liked, and I figured it was overkill to update every single time you initialize a new project. Doing it this way is no different than the way it works normally for a project: the modules are updated when the user runs npm update manually.

Reply:
Yeah.
Can you put something in the README to that effect?

Reply:
You mean saying that the user needs to update the packages manually?

Reply:
For reference, these are the init times I was seeing with the different approaches:
npm install in new project: ~8s
npm update before copy: ~5s
no update: ~2s